### ISSUE TITLE
Loklak /api/peers.json on the website says the count is 0

### ISSUE THREAD

**Issue by sudheesh001** [loklak/loklak_server#15]
Running the following query `http://loklak.org/api/peers.json` results in
```json
{
  "count" : "0",
  "peers" : [ {
    "host" : "80.152.219.115",
    "port.http" : -1,
    "port.https" : -1,
    "lastSeen" : 1429177934409,
    "lastPath" : "/api/search.json",
    "peername" : "anonymous"
  }, 
  .
  .
  .
```
Where as the actual count of the number of "peers" i.e. Array size in this case is `69`

**Comment by smokingwheels** [comment]
Ok so if you have labeled your peer it wont show on the list.

**Comment by Orbiter** [comment]
fixed in c4028faa9c5661a4549b02fd9cac581b85f850a7
